data/json/items/tool/fishing.json

```diff
@@ -82,5 +82,66 @@
     "use_action": [ "FISH_ROD" ],
     "qualities": [ [ "FISHING", 2 ] ],
     "flags": [ "FISH_GOOD", "SHEATH_SPEAR" ]
-  }
+  },
+  {
+  "id": "plastichoboreel",
+  "type": "TOOL",
+  "looks_like": "baton",
+  "//": "based on the ka-bar kaster",
+  "name": {
+    "str": "plastic hand fishing reel"
+  },
+  "description": "A commercial version of what is often known as a 'hobo fishing reel'.  It is a short rod with a section on one end with fishing line wrapped around it, with a rotating guard around that.  It is not as effective as a normal fishing pole, but is much more compact than them.",
+  "weight": "400 g",
+  "volume": "400 ml",
+  "longest_side": "158 mm",
+  "price": 1300,
+  "price_postapoc": 20000,
+  "to_hit": -1,
+  "bashing": 4,
+  "material": [
+    "plastic"
+  ],
+  "symbol": "/",
+  "color": "black",
+  "use_action": [
+    "FISH_ROD"
+  ],
+  "qualities": [
+    [ "FISHING", 1 ]
+  ],
+  "flags": [
+    "FISH_GOOD"
+  ]
+},
+  {
+  "id": "hoboreel",
+  "type": "TOOL",
+  "looks_like": "pointy_stick",
+  "name": {
+    "str": "hobo hand fishing reel"
+  },
+  "description": "What is often known as a 'hobo fishing reel'.  It is a short wooden rod with a section on one end with fishing line wrapped around it, with a notch to put the string to help pull in fish.  It is not as effective as a normal fishing pole, but is much more compact than them.",
+  "weight": "500 g",
+  "volume": "380 ml",
+  "longest_side": "150 mm",
```

Comment:
Were the units supposed to be centimeters?

Reply:
No, the hobo fishing reel is only about that long.